rpc_util.go

```diff
@@ -429,9 +429,9 @@ func (o ContentSubtypeCallOption) before(c *callInfo) error {
 }
 func (o ContentSubtypeCallOption) after(c *callInfo, attempt *csAttempt) {}
 
-// ForceCodec returns a CallOption that will set the given Codec to be
+// ForceCodec returns a CallOption that will set the given codec to be
```

Comment:
IntelliJ recognizes `Codec` as a reference to the deprecated type, which it is not. So I thought it's ok to change it to the reference to the formal parameter instead.